fe/src/components/RankAndStats/RankAndStats.re

```diff
@@ -51,17 +36,31 @@ let renderContent = (send, users, isUsersLoading) =>
 
 let make = _children => {
   ...component,
-  initialState,
-  reducer,
-  didMount: ({send}) => send(GetUsersSvc),
-  render: ({state, send}) =>
+  render: _self => {
+    let usersQuery = GetUsers.make();
+
     <div className="rankAndStats">
-      {
-        switch (state) {
-        | FAILURE => <FailureMask />
-        | LOADING => renderContent(send, [], true)
-        | LOADED(users) => renderContent(send, users, false)
-        }
-      }
-    </div>,
+      <GetUsersQuery
+        notifyOnNetworkStatusChange=true variables=usersQuery##variables>
+        ...{
+             ({result, refetch, networkStatus}) => {
+               let refreshUsers = refreshUsersWithRefetch(refetch);
+               let isRefeching = networkStatus == Some(4);
```

Comment:
Trzeba by wyjaśnić nieco co oznacza `Some(4)`.

Reply:
To się właśnie wiąże troche z powyższym pytaniem. Akurat Some(4) oznacza że dane pobierają się ponownie - tak mają w dokumentacji react-apollo. Wyniosłem to do stałej
Niewiem czy to co pisałem wyżej i tutaj jest zrozumiałe jak coś mogę potłumaczyć na żywo.